src/lib/openapi/spec/client-app-schema.ts

```diff
@@ -0,0 +1,63 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { parametersSchema } from './parameters-schema';
+
+export const clientAppSchema = {
+    $id: '#/components/schemas/clientAppSchema',
+    type: 'object',
+    required: ['appName', 'instanceId'],
+    properties: {
+        appName: {
+            type: 'string',
+        },
+        instanceId: {
+            type: 'string',
+        },
+        clientIp: {
+            type: 'string',
+        },
+        environment: {
+            type: 'string',
+        },
+        seenToggles: {
+            type: 'array',
+            items: {
+                type: 'string',
+            },
+        },
+        metricsCount: { type: 'number' },
+        strategies: {
+            type: 'array',
+            items: {
+                oneOf: [
+                    { type: 'string' },
+                    { $ref: '#/components/schemas/parametersSchema' },
```

Comment:
Could inline `parametersSchema` here since it's so small:

```ts
strategies: {
    type: 'array',
    items: {
        oneOf: [
            { type: 'string' },
            { type: 'object', additionalProperties: { type: 'string' } },
        ]
    }
},
```